README.md

```diff
@@ -41,7 +41,65 @@ This model provides large gas savings compared to previous relay designs (TODO:
 benchmarking). It also gets especially attractive if EIP2028 activates,
 reducing calldata gas costs.
 
-### Project Notes
+### A Note on Endianness
+
+Bitcoin internally uses little-endian representations of integers and digests.
+Block explorers and most user-facing applications use the more common
+big-endian representation. To minimize order swaps and prevent confusion, all
+our tooling uses the LE representation exclusively. If using the JS, rust,
+golang, or python tooling in [bitcoin-spv](http://bitcoin-spv.com), everything
+will Just Work. If writing custom software using data from block explorers,
+full nodes, or other data sources, make sure digests are LE before submitting
+to the relay.
```

Comment:
Do we have a function for reversing endianness in Solidity?  We could mention that here.

Reply:
they _should_ never need the solidity functions